src/style.css

```
@@ -1 +1,16 @@
-/* styles go here */
+.set {
+  margin-bottom: 20px;
+}
+
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.last {
+  margin-bottom: 0;
+}
```

not a blocking comment - you could do like this.
However a more generic approach that would save you from adding another class would be this:
```suggestion
.form-field:last-child {
  margin-bottom: 0;
}
```
so you just add as may inputs and you need and don't bother to remind yourself adding another style to the last element.